example/example_SG.md

```diff
@@ -0,0 +1,101 @@
+
+## Cultural references, idioms, and slang
+
+When dealing with cultural references, idioms, and slang in English, references to American pop culture (music, movies), sports (basketball, baseball, football), sayings, and some proverbs will be generally recognized. If a cultural reference is difficult to express in English, use the meaning to find a good equivalent. Refer to sources and databases of common expressions to help find an adequate equivalent.
+
+* [List of Common Proverbs](https://www.engvid.com/english-resource/50-common-proverbs-sayings/)
+* [List of Common Idioms](http://www.smart-words.org/quotes-sayings/idioms-meaning.html)
+
+## Local units
+
+### Date format
+
+| | Format | Example |
+| - | - | - |
+| Short | mm/dd/yy | 12/31/99 |
+| Abbreviated | mmm dd | Dec 31 |
+| Long | Month Date, Year | December 31, 1999 |
+
+It is acceptable to use any of the date formats; however, always coordinate the translated date format with the source text unless the specifications for a project indicate otherwise.
+
+### Calendar view
+
+In the United States, the 12-month Gregorian calendar is used with the week starting on Monday.
+
+### Time format
+
+Generally, a 12-hour clock is used in the United States. 24-hour clocks are used in the military and it is possible that 24-hour time could be used to make reference or allusion to military.
+
+### Numerals
+
+| Separator | Character Name | Symbol | Example |
+| - | - | - | - |
+| Decimal | Period | `.` | 1.23 |
+| Thousands | Comma | `,` | 1,234 |
+
+### Currency
+
+The currency of the United States is the dollar (USD). The monetary symbols are `$` and rarely `¢`.
+
+### Units of measurement
+
+The Imperial system is used for measurement in the United States.
+
+### Address and postal code format
+
+    [addressee]
+    [street number and name][building and suite numbers]
+    [district][city][state/province][postal code]
+    [Country]
+
+    Mozilla
+    331 East Evelyn Ave.
+    Mountain View, CA 94041
+    USA
+
+### Telephone number format
+
+Area codes are surrounded by parentheses and separated by a space. The Phone number is separated by a dash.
+
+`(###) ###-####`
+
+## Emphasis
+
+Read about emphasis styles in the [General Mozilla L10n Style Guide](general_L10n_SG.md#emphasis-guidelines).
+
+### Firefox for desktop
+
+Use title case for labels for items in the menu, navigation and section headings, page titles, and buttons.
+
+Use sentence case for headlines, which are the more conversational titles and subtitles that appear in instructional text screens, tours and form pages,  and text links without terminal punctuation unless the phrase is a question or exclamation or if the link is embedded in running text. (Note, include the terminal punctuation in hyperlink if the entire phrase is linked, and do not include the terminal punctuation in the hyperlink if the link is within running text)
```

Comment:
double space before *and text link*

The last sentence is missing a period, and should likely not be between parentheses.